### PR TITLE
Remove CLI from Core

### PR DESCRIPTION
**Pull request by antho1404**
I'm working on another CLI separated from the core and in JS here https://github.com/mesg-foundation/cli.

So we take this decision, all the files in this PR are not necessary anymore

### CODE REVIEW

File: dev-core [hunk 1]
```diff
@@ -78,18 +76,25 @@ if [[ ! $BINCACHED ]] || [[ ! $DOCKERCACHED ]]; then
   docker build -f Dockerfile.dev -t "mesg/core:$VERSION" .
 fi
 
-./bin/cli stop
+# docker service rm core
```
Comment (NicolasMahe) [inline dev-core]: forgotten comment here?

**Comment by antho1404** [comment]
yes can't find how to fix that but can be done in another PR if needed and anyway it's working and just for dev so it's fine to keep it like that

**Comment by NicolasMahe** [comment]
@krhubert please also review and test ;)

**Comment by NicolasMahe** [comment]
Let's merge https://github.com/mesg-foundation/engine/pull/974 so we can have a last working CLI compatble with single-output until the js version is ready!

**Comment by NicolasMahe** [comment]
There is a missing stuff in this PR:
- [ ] removing the CircleCi task about the CLI!

**Comment by NicolasMahe** [comment]
Another PR to remove some scripts: https://github.com/mesg-foundation/engine/pull/996

**Comment by NicolasMahe** [comment]
@mesg-foundation/core i will force merge this PR when test are green!

**Pull request merged**